cli/cli/out/file_logger.go

```diff
@@ -69,6 +123,7 @@ func setupFileLogger() error {
 		CallerPrettyfier:          nil,
 	}
 
+	// creates a files logger which logs data onto the log file
 	fileLogger = logrus.New()
 	fileLogger.SetOutput(logFile)
 	fileLogger.SetLevel(logrus.InfoLevel)
```

Comment:
is `logrus.InfoLevel` enough?

Reply:
yeah; `SetLevel` accepts the minimum required level .. so this means any level above than info will trigger hook .. so error will be included